src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/ServiceAccountProviderHelper.scala

```diff
@@ -46,15 +46,15 @@ class ServiceAccountProviderHelper(wrappedServiceAccountProvider: ServiceAccount
     try { future.recoverWith(exceptionHandler) } catch exceptionHandler
   }
 
-  override def getClusterServiceAccount(userInfo: UserInfo, googleProject: GoogleProject)(implicit executionContext: ExecutionContext): Future[Option[WorkbenchEmail]] = {
+  override def getClusterServiceAccount(workbenchEmail: WorkbenchEmail, googleProject: GoogleProject)(implicit executionContext: ExecutionContext): Future[Option[WorkbenchEmail]] = {
```

Comment:
Had to change these provider interfaces to take a `WorkbenchEmail` instead of a `UserInfo`. This is totally fine now that we're using `SwaggerSamClient` - we don't need to pass user tokens anymore!